inst/cpp_generation/model_input.csv

```diff
@@ -24,8 +24,8 @@ paed_cd4_dist,Children.hc1_cd4_dist,real_type,ModelVariant::run_child_model,,FAL
 paed_cd4_transition,Children.hc_cd4_transition,real_type,ModelVariant::run_child_model,,FALSE,2,children.hc2DS,children.hc1DS,,
 paed_cd4_mort,Children.hc1_cd4_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc1DS,children.hcTT,children.hc1AG,
 adol_cd4_mort,Children.hc2_cd4_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc2DS,children.hcTT,children.hc2AG,
-paed_cd4_prog,Children.hc1_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc1DS,,,
-adol_cd4_prog,Children.hc2_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc2DS,,,
+paed_cd4_prog,Children.hc1_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc1DS,2,base.NS,
+adol_cd4_prog,Children.hc2_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc2DS,1,base.NS,
```

Comment:
These are now hardcoded, what are the 2 and 1 in the 2nd dimension? Do we need a 1 would that not be the same as `children.hc2DS x base.NS`?

Reply:
Both of these represent the number of coarse age groups. For paed_cd4_prog, this is 0-2 year olds and 3-4 year olds. I've added variables for these now.